cmd/uhc/account/quota/cmd.go

```diff
@@ -0,0 +1,172 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package quota
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift-online/uhc-cli/pkg/dump"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	amv1 "github.com/openshift-online/uhc-sdk-go/pkg/client/accountsmgmt/v1"
+
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+	json  bool
+	org   string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "quota",
+	Short: "Retrieve cluster quota information.",
+	Long:  "Retrieve cluster quota information of a specific organization.",
+	Run:   run,
+}
+
+func init() {
+	// Add flags to rootCmd:
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+	flags.BoolVar(
+		&args.json,
+		"json",
+		false,
+		"Returns a list of resource quota objects in JSON.",
+	)
+	flags.StringVar(
+		&args.org,
+		"org",
+		"",
+		"Specify which organization to query information from. Default to local users organization.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) {
+
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't load config file: %v\n", err)
+		os.Exit(1)
+	}
+	if cfg == nil {
+		fmt.Fprintf(os.Stderr, "Not logged in, run the 'login' command\n")
+		os.Exit(1)
+	}
+
+	// Check that the configuration has credentials or tokens that haven't have expired:
+	armed, err := config.Armed(cfg)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't check if tokens have expired: %v\n", err)
+		os.Exit(1)
+	}
+	if !armed {
+		fmt.Fprintf(os.Stderr, "Tokens have expired, run the 'login' command\n")
+		os.Exit(1)
+	}
+
+	// Create the connection:
+	logger, err := util.NewLogger(args.debug)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't create logger: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Create the connection, and remember to close it:
+	connection, err := client.NewConnectionBuilder().
+		Logger(logger).
+		TokenURL(cfg.TokenURL).
+		Client(cfg.ClientID, cfg.ClientSecret).
+		Scopes(cfg.Scopes...).
+		URL(cfg.URL).
+		User(cfg.User, cfg.Password).
+		Tokens(cfg.AccessToken, cfg.RefreshToken).
+		Insecure(cfg.Insecure).
+		Build()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't create connection: %v\n", err)
+		os.Exit(1)
+	}
+	defer connection.Close()
+
+	// // Organization to search in case one was not provided:
+	if args.org == "" {
+		// Get organization of current user:
+		userConn, err := connection.AccountsMgmt().V1().CurrentAccount().Get().
+			Send()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Can't retrieve current user information: %v\n", err)
+			os.Exit(1)
+		}
+		userOrg, _ := userConn.Body().GetOrganization()
+		args.org = userOrg.ID()
+	}
+
+	// Get connection
+	orgRequest := connection.AccountsMgmt().V1().Organizations().Organization(args.org)
```

Comment:
The name `orgRequest` is a bit confusing. What you have here is not a request, but a client for the resource that manages the organization. For this kind of variables I'd suggest to use `orgResource` (or `orgCollection` when the resource is a collection).